src/clientdlg.cpp

```diff
@@ -274,6 +274,10 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     // File menu  --------------------------------------------------------------
     QMenu* pFileMenu = new QMenu ( tr ( "&File" ), this );
 
+    pFileMenu->addAction ( tr ( "&Connection Setup..." ), this, SLOT ( OnOpenConnectionSetupDialog() ), QKeySequence ( Qt::CTRL + Qt::Key_C ) );
```

Comment:
Ctrl-C means Cancel.  It means Cancel on Windows, Linux and MacOS CLI environments.  I'm hesitant about using it for anything else.  (It's bad having to remember Ctrl-Z - background - vs Ctrl-Z - undo...)

Personally, I'd use Ctrl-O (like "open" a new connection).  But I guess it was already Ctrl-C somewhere else...

Reply:
Yes it was Ctrl-C before - I left all the keyboard shortcuts exactly as they were.

I am planning to do a sequence of UI -related pull requests.  For the next pull request there are a few menu items I'd like to add, and some other small changes.  For me, that would be the ideal opportunity to make sure everyone is happy with the keyboard shortcuts and the 'menu accelerators'.

I believe the menu accelerator keys change according to the user's language, but the keyboard shortcuts always remain the same - is that right ?